Topic message acceptance test fails in previewnet

### Description

Topic message acceptance test failed in previewnet when validating v0.130.0-rc1 deployment

```
  JUnit Platform Suite:AcceptanceTest:Cucumber:HCS Base Coverage Feature:Validate Topic message submission:Examples:Example #1.1
    ClasspathResourceSource [classpathResourceName = 'features/topic/topic.feature', filePosition = FilePosition [line = 30, column = 7]]
    => java.lang.reflect.UndeclaredThrowableException
       org.hiero.mirror.test.e2e.acceptance.steps.TopicFeature$$SpringCGLIB$$0.submitTopicMessage(<generated>)
       java.base/java.lang.reflect.Method.invoke(Unknown Source)
       io.cucumber.java.Invoker.doInvoke(Invoker.java:66)
       io.cucumber.java.Invoker.invoke(Invoker.java:24)
       io.cucumber.java.AbstractGlueDefinition.invokeMethod(AbstractGlueDefinition.java:47)
       [...]
     Caused by: com.hedera.hashgraph.sdk.PrecheckStatusException: Hedera transaction `0.0.8035@1747178837.784000586` failed pre-check with the status `INSUFFICIENT_PAYER_BALANCE`
       com.hedera.hashgraph.sdk.Executable$GrpcRequest.mapStatusException(Executable.java:962)
       com.hedera.hashgraph.sdk.Executable.execute(Executable.java:449)
       com.hedera.hashgraph.sdk.Transaction.execute(Transaction.java:36)
       com.hedera.hashgraph.sdk.ChunkedTransaction.executeAll(ChunkedTransaction.java:317)
       com.hedera.hashgraph.sdk.TopicMessageSubmitTransaction.executeAll(TopicMessageSubmitTransaction.java:30)
       [...]
```

### Steps to reproduce

Run acceptance tests against previewnet with cucumber tag `@topic and @release`

### Additional context

_No response_

### Hedera network

previewnet

### Version

v0.130.0-rc1

### Operating system

None

--- a/test/src/test/java/org/hiero/mirror/test/e2e/acceptance/client/AccountClient.java
+++ b/test/src/test/java/org/hiero/mirror/test/e2e/acceptance/client/AccountClient.java
@@ -36,7 +36,7 @@ import org.springframework.retry.support.RetryTemplate;
 @Named
 public class AccountClient extends AbstractNetworkClient {
 
-    private static final long DEFAULT_INITIAL_BALANCE = 50_000_000L; // 0.5 ℏ
+    private static final long DEFAULT_INITIAL_BALANCE = 75_000_000L; // 0.75 ℏ
 
     private final Map<AccountNameEnum, ExpandedAccountId> accountMap = new ConcurrentHashMap<>();
     private final Collection<ExpandedAccountId> accountIds = new CopyOnWriteArrayList<>();

